src/main/resources/storyquestions/article/index.fjs

```diff
@@ -1 +1,164 @@
-//@flow
\ No newline at end of file
+//@flow
+
+import { Actions, ComponentTypes } from 'lib/ophan';
+import { fromEvent } from 'lib/events';
+import type { Channel } from 'channels';
+
+type QuestionId = string;
+type QuestionText = string;
+type Question = [?QuestionId, ?QuestionText];
```

Comment:
why did you choose an array here, rather than an object?

Reply:
Flow has support for "tuples" as lists of n-items: https://flow.org/en/docs/types/tuples/